metric/system/process/process_windows.go

```diff
@@ -351,3 +385,49 @@ func getProcCredName(pid int) (string, error) {
 
 	return fmt.Sprintf(`%s\%s`, domain, account), nil
 }
+
+func getIdleProcessTime() (float64, float64) {
+	idle, kernel, user, err := gowindows.GetSystemTimes()
+
+	// Average by cpu because GetSystemTimes returns summation of across all cpus
+	num_cpus := float64(runtime.NumCPU())
+	idleTime := float64(idle) / num_cpus
+	kernelTime := float64(kernel) / num_cpus
+	userTime := float64(user) / num_cpus
+
+	if err != nil {
+		return 0, 0
+	}
+	// Calculate total CPU time, averaged by cpu
+	totalTime := idleTime + kernelTime + userTime
```

Comment:
I'm not sure if this is right.  The following doc makes me believe that kernelTime includes idleTime.  Did you find this to be different in practice?

 https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-getsystemtimes#parameters

Reply:
You're correct.kernelTime includes idleTime If I directly use the win32 api.

But as you can see here, I use `https://github.com/elastic/go-windows`
 https://github.com/elastic/elastic-agent-system-metrics/blob/5e3c5cf781aadb14be48e4ef2216e292b77fcdd2/metric/system/process/process_windows.go#L390

 and that takes the above thing into the account and calculates kernelTime as `kernelTime - idleTime` before returning the values. See [here](https://github.com/elastic/go-windows/blob/ee94ac30503b3ecb77e35710f8648d500919f3b9/kernel32.go#L192-L204)

Reply:
The results of this function are accurate with the task manager's stats.

Reply:
Great, Thanks.